src/django_perf_rec/db.py

```diff
@@ -11,7 +12,9 @@
 
 class DBOp(Operation):
     def __repr__(self):
-        return "DBOp({!r}, {!r})".format(repr(self.alias), repr(self.query))
+        return "DBOp({!r}, {!r}, {!r})".format(
+            repr(self.alias), repr(self.query), repr(self.tb)
+        )
```

Comment:
Thanks for updating this, but I found it weird this was the only class with a `__repr__`. I couldn't find use or tests, so have removed it in #315.